spec/services/virtual_hearings/link_service_spec.rb

```diff
@@ -49,21 +49,46 @@
         allow(ENV).to receive(:[]).with("VIRTUAL_HEARING_PIN_KEY").and_return PIN_KEY
         allow(ENV).to receive(:[]).with("VIRTUAL_HEARING_URL_HOST").and_return URL_HOST
         allow(ENV).to receive(:[]).with("VIRTUAL_HEARING_URL_PATH").and_return URL_PATH
-        allow(VirtualHearings::SequenceConferenceId).to receive(:next).and_return "0000001"
       end
 
-      it "returns the expected URL", :aggregate_failures do
-        uri = URI(described_class.new.host_link)
-        expect(uri.scheme).to eq "https"
-        expect(uri.host).to eq URL_HOST
-        expect(uri.path).to eq "#{URL_PATH}/"
-
-        query = Hash[*uri.query.split(/&|=/)]
-        expect(query["conference"]).to eq "BVA0000001@#{URL_HOST}"
-        expect(query["name"]).to eq VirtualHearings::LinkService::JUDGE_NAME
-        expect(query["pin"]).to eq "3998472"
-        expect(query["callType"]).to eq "video"
-        expect(query["join"]).to eq "1"
+      context "the sequence returns '0000001'" do
+        before do
+          allow(VirtualHearings::SequenceConferenceId).to receive(:next).and_return "0000001"
+        end
+
+        it "returns the expected URL", :aggregate_failures do
+          uri = URI(described_class.new.host_link)
+          expect(uri.scheme).to eq "https"
+          expect(uri.host).to eq URL_HOST
+          expect(uri.path).to eq "#{URL_PATH}/"
+
+          query = Hash[*uri.query.split(/&|=/)]
```

Comment:
```suggestion
          query = Rack::Utils.parse_query(uri.query)
```
Minor, and possible not worth it? The current code is fully understandable without reading docs.

Reply:
addressed in 4f6acac